<?xml version="1.0"?>
<launch>

    <rosparam file="$(find graph_slam)/config/intrinsics.yaml"/>

    <arg name="in_odom_topic" default="/odom/"/>
    <arg name="in_depthmap_topic" default="/depthmap/"/>
    <arg name="out_cloud_topic" default="/cloud_synced/"/> 
    <arg name="subsample_factor" default="1"/>
   
    <node pkg="graph_slam" name="sync_odom_camera_node" type="sync_odom_camera_node" output="screen">
        <param name="in_odom_topic" value="/odom/"/>
        <param name="in_depthmap_topic" value="/depthmap/"/>
        <param name="out_cloud_topic" value="/cloud_synced/"/>
        <param name="subsample_factor" value="10"/>
    </node>

</launch>
